### PR TITLE
Removes our cryo mirror

### PR DESCRIPTION
**Pull request by ghost**
It's not needed, and it in fact delays us getting fixes for cryo-related issues

### CODE REVIEW

File: hippiestation/code/game/machinery/cryo.dm [hunk 1]
```diff
@@ -1,391 +0,0 @@
...
-	max_integrity = 350
-	armor = list(melee = 0, bullet = 0, laser = 0, energy = 100, bomb = 0, bio = 100, rad = 100, fire = 30, acid = 30)
-	layer = ABOVE_WINDOW_LAYER
-
-	var/on = FALSE
-	state_open = FALSE
-	var/volume = 100
-
-	var/efficiency = 1
-	var/sleep_factor = 750
```
Comment (GuyonBroadway) [inline hippiestation/code/game/machinery/cryo.dm:19]: actually we need to set this to 0 else people are put to sleep.

File: hippiestation/code/game/machinery/cryo.dm [hunk 2]
```diff
@@ -1,391 +0,0 @@
...
-	var/paralyze_factor = 1000
```
Comment (GuyonBroadway) [inline hippiestation/code/game/machinery/cryo.dm:20]: This also needs to be set to 0

File: hippiestation/code/game/machinery/cryo.dm [hunk 3]
```diff
@@ -1,391 +0,0 @@
...
-			L.update_canmove()
-	occupant = null
-	update_icon()
-
-/obj/machinery/atmospherics/components/unary/cryo_cell/close_machine(mob/living/carbon/user)
-	if((isnull(user) || istype(user)) && state_open && !panel_open)
-		..(user)
-		return occupant
-
-/obj/machinery/atmospherics/components/unary/cryo_cell/container_resist(mob/living/user)
```
Comment (GuyonBroadway) [inline hippiestation/code/game/machinery/cryo.dm]: We also need to mirror this proc.

File: hippiestation/code/game/machinery/cryo.dm (hunk 1 above)
Comment (ghost) [reply hippiestation/code/game/machinery/cryo.dm:19]: done

File: hippiestation/code/game/machinery/cryo.dm (hunk 2 above)
Comment (ghost) [reply hippiestation/code/game/machinery/cryo.dm:20]: done

**Review by ghost — DISMISSED** [review]

**Comment by ghost** [comment]
Good change

**Comment by ghost** [comment]
nice

**Review by ghost — APPROVED** [review]

**Pull request merged**